### ISSUE TITLE
ChainedManyToManyField value not selected after saving data

### ISSUE THREAD

**Issue by devrijal** [digi604/django-smart-selects#112]
Hi , many thanks for the app, but I have some problem, when I select the data, then save, it seem working correctly, when I review the current saved data, ChainedManyToManyField value not selected, what's the problem ?

``` python
category = ManyToManyField(Category, blank=True)

subcategory = ChainedManyToManyField(
        Subcategory,
        help_text="Subcategory",
        chained_field="category",
        chained_model_field="category", blank=True)
```
I think there is no problem with my code because it saving the data correctly.

any explanation will appreciated :+1:

**Comment by bgarcial** [comment]
@devrijal I also have the same inconvenient [https://github.com/digi604/django-smart-selects/issues/122](url)

You will can solve this situation? 
Thanks :smile:

**Comment by bgarcial** [comment]
@digi604 and @millin  do you know that can be happenin about it? 

PD: Excuse me for metions to you, may be can support us. :smile:

**Comment by rantecki** [comment]
Just ran into this issue myself.  See patch here: https://github.com/digi604/django-smart-selects/pull/139

**Comment by blag** [comment]
Since this is now merged in I'm closing this issue. Feel free to reopen if it's not solved by #139.
